src/App.vue

```diff
@@ -111,37 +115,75 @@ export default {
         case 'left':
           targetRow = this.randomAvailbleRow()
           if (targetRow) {
-            this.tiles.push({x: 0, y: targetRow, number: this.pickRandomTileNumber()})
+            this.tiles.push({x: 0, y: targetRow, number: this.pickRandomTileNumber(), id: Math.random()})
           }
           break;
         case 'right':
           targetRow = this.randomAvailbleRow()
           if (targetRow) {
-            this.tiles.push({x: 5, y: targetRow, number: this.pickRandomTileNumber()})
+            this.tiles.push({x: 5, y: targetRow, number: this.pickRandomTileNumber(), id: Math.random()})
           }
           break;
         case 'top':
           targetCol = this.randomAvailbleCol()
           if (targetCol) {
-            this.tiles.push({x: targetCol, y: 0, number: this.pickRandomTileNumber()})
+            this.tiles.push({x: targetCol, y: 0, number: this.pickRandomTileNumber(), id: Math.random()})
           }
           break;
         case 'bottom':
           targetCol = this.randomAvailbleCol()
           if (targetCol) {
-            this.tiles.push({x: targetCol, y: 5, number: this.pickRandomTileNumber()})
+            this.tiles.push({x: targetCol, y: 5, number: this.pickRandomTileNumber(), id: Math.random()})
           }
           break;
         default:
 
       }
     },
 
+    highestNumber() {
+      return this.tiles.reduce((accumulator, tile) => {
+        return tile.number > accumulator ? tile.number : accumulator
+      }, 0)
+    },
+
     pickRandomTileNumber() {
```

Comment:
This function is doing too much work. If the function is called `pickRandomTileNumber` as a reader of the code I expect the function to only `pick` a tile, and return it. But looks like it is doing a lot of things, and all of that extra stuff shouldn't belong to this function. Maybe extract all that out into another function called `doStuff`. And at the caller of `pickRandomTileNumber`, you will also have to call the `doStuff` function.